### PR TITLE
Adding support for Octeon DPU family

### PR DESCRIPTION
**Pull request by navadiaev**
Issue was opened per instructions:

https://github.com/k8snetworkplumbingwg/sriov-network-operator/issues/355

### CODE REVIEW

File: doc/vendor-support.md [hunk 1]
```diff
@@ -12,4 +12,4 @@ as `active-in-project` in the table below.
 | Nvidia(Mellanox) | active-in-project |
 | Intel | active-in-project |
 | HPE(Marvell Qlogic) | BSN_dev_support@hpe.com |
-
+| Marvell | active-in-project |
```
[the remark this answers is not in the document]
Comment (naftalyava) [reply doc/vendor-support.md:15]: Hi Adrian,

I am an engineer working for Marvell.
We are looking to be involved in this project from now on.

[the remark this answers is not in the document]
Comment (navadiaev) [reply doc/vendor-support.md:15]: Hi Adrian,

I am an engineer working for Marvell.
We are looking to be involved in this project from now on.

[the remark this answers is not in the document]
Comment (adrianchiris) [reply doc/vendor-support.md:15]: ack, glad to have you on board !

**Review by adrianchiris — APPROVED** [review]

File: doc/vendor-support.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SchSeba) [reply doc/vendor-support.md:15]: Until there is a Marvell maintainer can you please add a contact email?
Then we can remove it when we promote a marvell developer to a maintainer

[the remark this answers is not in the document]
Comment (navadiaev) [reply doc/vendor-support.md]: done.
Can we push this now?

**Comment by rollandf** [comment]
@navadiaev Can you please squash the commits into a single one?

**Comment by navadiaev** [comment]
> 

Done.
Sorry, I was thinking PR squash commit automatticaly...

**Comment by navadiaev** [comment]
Hi anything else we need to do to merge it?

**Review by e0ne — APPROVED** [review]

**Review by SchSeba — APPROVED** [review]

**Pull request merged**